include/psa/crypto_extra.h

```diff
@@ -121,6 +121,14 @@ psa_status_t mbedtls_psa_inject_entropy(const unsigned char *seed,
                                         size_t seed_size);
 
 
+/* The Number of key slots (plus one because 0 is not used).
+ * The value is a compile-time constant for now, for simplicity. */
+#if defined(MBEDTLS_PSA_KEY_SLOT_COUNT)
+#else
+    #define MBEDTLS_PSA_KEY_SLOT_COUNT 32
```

Comment:
i understand this is the naming convention for external define.
but still mbedtls has nothing to do with psa key slots number.
can we not have the mbedtls prefix? have some other?
why not leave it PSA_KEY_SLOT_COUNT?

Reply:
I asked @gilles-peskine-arm and @Patater  on their inputs on this and this is the name @gilles-peskine-arm requested.

Reply:
slot count is public define now, so it can be PSA_xxx ...

Reply:
I think @gilles-peskine-arm said that PSA_XXX is only for public PSA API and this is not as it is a configuration for MbedCrypto that other implementations do not currently have to support.